src/glados/tests/test_comp_rep_card.py

```diff
@@ -80,10 +80,6 @@ def test_compound_report_card_scenario_1(self):
     # Compound Name and Classification
     # --------------------------------------
 
-    # Normal structure image
```

Comment:
The tests for the images are already in http://chembl-glados.herokuapp.com/js_tests/?spec=Compound
This selenium tests fail randomly and it's really annoying.